README.md

```diff
@@ -1,2 +1,75 @@
-# baby-bliss-bot
-An exploratory research to generate new Bliss vocabulary
+# Baby Bliss Bot
+
+An exploratory research to generate new Bliss vocabulary.
```

Comment:
Minor typo: An exploratory research *project*

Perhaps also "to generate new Bliss vocabulatory using machine learning techniques." or something like that?